capycli/project/create_project.py

```diff
@@ -95,6 +95,10 @@ def update_project(self, project_id: str, project: Optional[Dict[str, Any]],
             if project_info and element in project_info:
                 project_info.pop(element)
 
+        # add information that this project was created by CaPyCli
+        project_info["additionalData"] = {}
+        project_info["additionalData"]["createdWith"] = "CaPyCli: " + capycli.get_app_version()
```

Comment:
As written above, I unfortunately have no time to test this now, but this looks risky. Wouldn't this always (a) add this info to projects we update - which is a different behaviour from what happens when updating releases and components where we only add the signature when creating new ones. Also, I think it could (b) overwrite all existing additionalData if the key is present in the REST PATCH body?

Reply:
Perfect, thanks!